presentation/src/main/java/com/knocklock/presentation/widget/Clock.kt

```diff
@@ -0,0 +1,69 @@
+package com.knocklock.presentation.widget
+
+import android.graphics.Color
+import android.widget.TextClock
+import androidx.compose.foundation.layout.Column
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.viewinterop.AndroidView
+import com.knocklock.presentation.util.TimeFormat
+import com.knocklock.presentation.util.TimeVerticalFormat
+import com.knocklock.presentation.util.TimeWithNoSecondFormat
+import com.knocklock.presentation.util.TimeWithSecondFormat
+
+
+@Composable
+fun ClockWidget(
+    modifier: Modifier = Modifier,
+    timeFormat: TimeFormat,
+) {
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        AndroidView(
+            factory = { context ->
+                TextClock(context).apply {
+                    format12Hour = timeFormat.timeFormat
+                    textSize = 36F
+                    setTextColor(Color.WHITE)
+                }
+            })
```

Comment:
Composable에 timeFormat이 변경되면 AndroidView도 업데이트 되어야 하니 update 람다를 구현해야 할 것 같아요!
아래 minute이랑 date clock도 부탁드려요~!

```suggestion
         AndroidView(
            factory = { context ->
                TextClock(context).apply {
                    format12Hour = timeFormat.timeFormat
                    textSize = 36F
                    setTextColor(Color.WHITE)
                }
            },
            update = { textClock ->
                textClock.format12Hour = timeFormat.timeFormat
            }
        )
```

Reply:
update 람다라는게 있군요. 반영하겠습니다 ^^